Pod/Classes/Cards/Expiry.swift

```diff
@@ -8,28 +8,28 @@
 
 import Foundation
 
-private let gregorianCalendar = NSCalendar(calendarIdentifier: NSCalendarIdentifierGregorian)!
+private let gregorianCalendar = Calendar(identifier: Calendar.Identifier.gregorian)
 
 /**
  A Credit Card Expiry date.
  */
 public struct Expiry: RawRepresentable {
 
     /// An invalid expiry date. This date is set to 1/1/1970 and therefor will be shown as `expired`.
-    public static let invalid = Expiry(rawValue: NSDate(timeIntervalSince1970: 0))
+    public static let invalid = Expiry(rawValue: Date(timeIntervalSince1970: 0))
 
-    public typealias RawValue = NSDate
+    public typealias RawValue = Date
 
-    public let rawValue: NSDate
+    public let rawValue: Date
 
     /// The month of the expiration date.
     public var month: UInt {
-        return UInt(components().month)
+        return UInt(components().month!)
```

Comment:
Good question. On second thoughts, in this case it should actually be fine, since these values are created from a Date object which always has both month and year.